packages/ui-extensions/src/surfaces/customer-account/targets.ts

```diff
@@ -90,6 +90,11 @@ export interface OrderStatusExtensionTargets {
       StandardApi<'customer-account.order-status.cart-line-list.render-after'>,
     AnyComponent
   >;
+  'customer-account.order.page.render': RenderExtension<
+    OrderStatusApi<'customer-account.order-status.cart-line-list.render-after'> &
+      StandardApi<'customer-account.order-status.cart-line-list.render-after'>,
```

Comment:
For both `OrderStatusApi<T>` and `StandardApi<T>`, shouldn't T = `customer-account.order.page.render` instead of `customer-account.order-status.cart-line-list.render-after`?

Reply:
damn... I updated locally, never pushed 🤦